src/components/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+import { FC } from 'react';
+
+export type Props = {
+  to: string;
+  text: string;
+  sex?: string;
+};
+
+export const PersonLink: FC<Props> = ({ to, text, sex }) => (
+  <NavLink
+    to={`../${to}`}
```

Comment:
Use a relative path instead of adding '../' to the 'to' prop. Replace with 'to={to}'.